app/presenters/organisations/show_presenter.rb

```diff
@@ -1,5 +1,9 @@
 module Organisations
   class ShowPresenter
+    include ActionView::Helpers::TagHelper
+    include ActionView::Helpers::UrlHelper
+    include ActionView::Helpers::TranslationHelper
```

Comment:
Maybe I'm missing something - what is the TranslationHelper needed for?

Reply:
It's for link 94 `t('organisations.foi.freedom_of_information_requests')`

Reply:
If you don't want to pull in another helper you can use `I18n.t('...')` without needing that, but works in the same way 👍